Dockerfile

```diff
@@ -5,8 +5,11 @@ COPY ui ./
 RUN npm install && \
     node_modules/.bin/ng build --prod
 
+ARG BASE_IMAGE_PREFIX
+FROM ${BASE_IMAGE_PREFIX}python:3.8-alpine
 
-FROM python:3.8-alpine
+ARG ARCH
+COPY qemu-${ARCH}-static* /usr/bin

```

Comment:
This will break the simple use case of checking out the code and running "docker build", for local development. I wonder if there's any solution possible to still support this use case.